core/src/main/java/org/stellar/anchor/sep10/JwtToken.java

```diff
@@ -51,6 +53,14 @@ public static JwtToken of(
     token.exp = exp;
     token.jti = jti;
     token.clientDomain = clientDomain;
+    String[] subs = sub.split(":", 2);
+    if (subs.length == 2) {
+      token.account = subs[0];
+      token.accountMemo = subs[1];
+    } else {
+      token.account = sub;
+      token.accountMemo = null;
+    }
```

Comment:
I was actually debating about this. Instead of throwing an error, the strategy is fall-back to non-splitting and let the downstream processing to decide whether we should throw an error. 

I suggest we can leave the fall-back strategy for the JWT token and let the downstream to throw "Invalid account" exception.